scripts/wdio/wdio-runner.js

```diff
@@ -0,0 +1,49 @@
+const Launcher = require('webdriverio').Launcher;
+
+async function wdioRunner(options) {
+  const {
+    configPath, locales, formFactors, useSeleniumGrid, continueOnFail, ...testSetup
+  } = options;
+  const factors = formFactors || [undefined];
+
+  process.env.USE_SELENIUM_GRID = useSeleniumGrid;
+
+  for (let localeI = 0; localeI < locales.length; localeI += 1) {
+    const locale = locales[localeI];
+    process.env.LOCALE = locale;
+
+    for (let factor = 0; factor < factors.length; factor += 1) {
+      let envValues = `LOCALE=${locale} `;
+
+      const form = factors[factor];
+      if (form) {
+        process.env.FORM_FACTOR = form;
+        envValues += `FORM_FACTOR=${form}`;
+      }
+
+      // eslint-disable-next-line no-console
```

Comment:
No, we should target this so that we're very explicit about when we're disabling eslint. I don't want the next guys to edit this file to just add console logs every where.